cpp/modmesh/universe/bezier.hpp

```diff
@@ -110,8 +109,181 @@ class Vector3d
 using Vector3dFp32 = Vector3d<float>;
 using Vector3dFp64 = Vector3d<double>;
 
+template <typename T>
+class PointPad
```

Comment:
`PointPad` is an expandable columnar container for points

Reply:
This is my first time to seeing the name `PointPad` instead of `PointCloud`.  Is there any reason behind this choice?

Reply:
Point cloud is a common term for a set of points without order. Pad is not an established name I would like for the specific type of container.